04/app.js

```diff
@@ -13,52 +13,44 @@ if(navElement) {
 const ulElement = document.createElement ('ul');
 navElement.appendChild(ulElement);
 
-for (let i = 0; i < 3; i++) {
+for (let i = 0; i <3; i++) {
```

Comment:
Ta pętla jest zbędna - lepiej wykorzystać forEach, o którym pisze poniżej.
Wtedy jej zawartośc tj. m. in. `createElement` będzie właśnie wew. forEach()